.flowconfig

```diff
@@ -23,6 +23,9 @@ node_modules/warning/.*
 ; Flow doesn't support platforms
 .*/Libraries/Utilities/HMRLoadingView.js
 
+; Flow doesn't support platforms
+.*/Libraries/Utilities/LoadingView.js
```

Comment:
Ah that's how you get around react itself throwing errors!

Reply:
Yeah, in this case is flow's inability to understand platform imports, `.ios` and `.android`, so we need to ignore this file.